apps/api/src/app/workflows-v2/shared/schemas/email-control.schema.ts

```diff
@@ -28,3 +31,8 @@ export const emailStepUiSchema: UiSchema = {
     skip: skipControl.uiSchema.properties.skip,
   },
 };
+
+export const emailStepControl = {
```

Comment:
Nit-pick comment: It's a bit of an overstretch to re-export as an object. It also adds an extra level of nesting when we import. Why was it necessary?